make_queue/tests/views/test_machine_views.py

```diff
@@ -74,3 +78,32 @@ def create_machine(name_prefix: str, machine_type_: MachineType, priority: Union
         for machine_type, correct_machine_order in zip(machine_types, correct_machine_orders):
             with self.subTest(machine_type=machine_type):
                 self.assertListEqual(list(machine_type.existing_machines), correct_machine_order)
+
+class CreateAndEditMachineViewTest(TestCase):
+
+    def setUp(self):
+        username = 'TEST_USER'
+        password = 'TEST_PASS'
+        self.user = User.objects.create_user(username=username, password=password)
+        change_permission = Permission.objects.get(codename="change_machine")
+        create_permission = Permission.objects.get(codename="add_machine")
+        self.user.user_permissions.add(create_permission, change_permission)
+        self.client.login(username=username, password=password)
+
+    def test_editMachine_context_data_has_EditMachineForm(self):
+        printer_machine_type = MachineType.objects.get(pk=1)
+        machine = Machine.objects.create(
+            name="Test",
+            machine_model="Ultimaker 2+",
+            machine_type=printer_machine_type
+        )
+        self.response = self.client.get(reverse("edit_machine", args=[machine.pk]))
+        
+        self.assertEqual(self.response.status_code, 200)
+        self.assertTrue(isinstance(self.response.context_data["form"], EditMachineForm))
+
+    def test_createMachine_context_data_has_baseMachineForm(self):
+        self.response = self.client.get(reverse("create_machine"))
+        
+        self.assertEqual(self.response.status_code, 200)
+        self.assertTrue(isinstance(self.response.context_data["form"], BaseMachineForm))
```

Comment:
Initially there were some more tests here that checked that `stream_name` is disabled. But as that was solved with javascript, it can't be tested here. The remaining tests are not that useful, so I am wondering if I should just remove them altogether?